scripts/appcenter-download-release.sh

```diff
@@ -8,53 +8,68 @@ APPCENTER_ENVIRONMENT="$1"
 APPCENTER_PLATFORM="$2"
 TARGET_VERSION="$3"
 
+INSTALL="${INSTALL:-"true"}"
```

Comment:
Some fix have been written for android, and a possibility to download ipa for iOS.

Despite it is not possible to use ipa within github action, I improved the script here to also work on macOS